packages/paste-core/components/chat-log/stories/components/ChatAttachment.stories.tsx

```diff
@@ -89,6 +89,17 @@ export const ComposerAttachmentCardExample: Story = () => (
         <ChatAttachmentDescription>123 MB</ChatAttachmentDescription>
       </ChatAttachment>
     </ComposerAttachmentCard>
+  </Stack>
+);
+
+export const StatefulComposerAttachmentCard: Story = () => (
```

Comment:
Note: just to make the VRT work better, I separated out the animating Attachment card into its own story that isn't checked by Chromatic.